### PR TITLE
Updated Docker

### PR DESCRIPTION
**Pull request by brylor**
Updated Docker to use nvm

### CODE REVIEW

**Comment by ar4s** [comment]
Replaced by #2751

**Comment by ar4s** [comment]
@brylor please specify version by adding `ENV NODE_VERSION 4.5.0` to `Dockerfile`

File: docker/provision.sh [hunk 1]
```diff
@@ -12,5 +12,5 @@ apt-get install -y --force-yes \
     python3.4 \
     python3.4-dev \
     python3-pip \
-    nodejs \
-    npm
+    wget \
+    vim
```
Comment (szok) [inline docker/provision.sh:16]: Why install Vim?

**Comment by szok** [comment]
:ok:

**Comment by ar4s** [comment]
:ok:

**Pull request merged**